docs/_includes/layouts/page.html

```diff
@@ -9,7 +9,7 @@
     <div class="d-flex mx-auto w100 wmx12">
         {% include 'navigation.html' %}
 
-        <main id="content" class="d-flex fl-grow1 ps-relative t64 py24 pl48 md:pl24 sm:pl16 sm:pr16">
+        <main id="content" class="d-flex fl-grow1 ps-relative pt96 pb24 pl48 md:pl24 sm:pl16 sm:pr16">
```

Comment:
@dancormier Forgot to mention that this change is necessary to make sure the #content anchor link doesn't end up being covered by the topbar. It slightly push the title of the page down but I think it's acceptable. Alternatively we could try pt64 and t24 to recreate the original space. It won't be perfect when the user click on skip to main content but it will still be better than it was before with a significant margin.